src/backend/TrafficCourts/Workflow.Service/Consumers/DisputantEmailSentConsumer.cs

```diff
@@ -23,10 +24,24 @@ public DisputantEmailSentConsumer(IOracleDataApiService oracleDataApiService, IL
         public async Task Consume(ConsumeContext<DisputantEmailSent> context)
         {
             using var scope = _logger.BeginConsumeScope(context);
+            _logger.LogDebug("Calling email sent service");
 
-            EmailHistory emailHistory = Mapper.ToEmailHistory(context.Message);
+            try
+            {
+                EmailHistory emailHistory = Mapper.ToEmailHistory(context.Message);
 
-            await _oracleDataApiService.CreateEmailHistoryAsync(emailHistory, context.CancellationToken);
+                await _oracleDataApiService.CreateEmailHistoryAsync(emailHistory, context.CancellationToken);
+            } 
+            catch (WebException ex)
+            {
+                HttpWebResponse? errorResponse = ex.Response as HttpWebResponse;
+                if (errorResponse?.StatusCode == HttpStatusCode.NotFound || errorResponse?.StatusCode == HttpStatusCode.BadRequest)
+                {
+                    // dont requeue if dispute not found or bad request
+                    _logger.LogError(ex.Message, context);
```

Comment:
This is incorrect usage of this logging method, the signature is `logger.LogError(string? message, params object?[] args)`. There is no format string being passed. Also, do not log `context`. This is a large object.  If you need specific properties, pull them out and add as properties.